nuitka/plugins/standard/config.yml

```diff
@@ -0,0 +1,1088 @@
+- "module-name": "boto3"
```

Comment:
Can we please not quote keywords, quotes sjhould only be applied to string values.